From 9ce44a3cd7a459a5eed00726e7746d49177e7fea Mon Sep 17 00:00:00 2001
From: zrj <rimvydas.jasinskas@gmail.com>
Date: Sat, 24 Aug 2019 15:45:46 +0300
Subject: [PATCH 46/52] <cpu/param.h>: Change visibility of several macros.

 While there, remove two unused ones since introduced in
 c8fe38ae0761c8117b13caf9b76ad5fb86ac2135
---
 sys/cpu/x86_64/include/param.h | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/sys/cpu/x86_64/include/param.h b/sys/cpu/x86_64/include/param.h
index e7b4c6dbc7..b55565847d 100644
--- a/sys/cpu/x86_64/include/param.h
+++ b/sys/cpu/x86_64/include/param.h
@@ -197,18 +197,16 @@
  */
 #define	round_page(x)	((((unsigned long)(x)) + PAGE_MASK) & ~(unsigned long)(PAGE_MASK))
 #define	trunc_page(x)	((unsigned long)(x) & ~(unsigned long)(PAGE_MASK))
-#define trunc_2mpage(x)	((unsigned long)(x) & ~(unsigned long)PDRMASK)
-#define round_2mpage(x)	((((unsigned long)(x)) + PDRMASK) & ~(unsigned long)PDRMASK)
 
 #if defined(_KERNEL) || defined(_KERNEL_STRUCTURES)
 #define	atop(x)		((vm_pindex_t)((x) >> PAGE_SHIFT))
-#endif
 #define	ptoa(x)		((vm_paddr_t)(x) << PAGE_SHIFT)
 
 #define	x86_64_btop(x)	((vm_pindex_t)((x) >> PAGE_SHIFT))
 #define	x86_64_ptob(x)	((vm_paddr_t)(x) << PAGE_SHIFT)
 
 #define	pgtok(x)		((x) * (PAGE_SIZE / 1024))
+#endif
 
 #ifdef _KERNEL
 
-- 
2.22.0

